### ISSUE TITLE
SFTP connection with incorrect password does not log the error

### ISSUE THREAD

**Issue by clmerriman** [proftpd/proftpd#1129]
The "pseudo" log entry for PASS is no longer appearing if an incorrect password is entered.  If a valid password is entered, the entry appears (as before).

It seems POST_CMR_ERR  and LOG_CMD_ERR are not being dispatched.

Applicable log excerpts:
1.3.5a
```
2020-09-21 15:09:51,954 hostname proftpd[5386] 192.168.100.112 (192.168.100.1[192.168.100.1]): dispatching POST_CMD command 'USERAUTH_REQUEST' to mod_sql
2020-09-21 15:09:51,954 hostname proftpd[5386] 192.168.100.112 (192.168.100.1[192.168.100.1]): dispatching LOG_CMD command 'USERAUTH_REQUEST' to mod_sql
2020-09-21 15:09:51,954 hostname proftpd[5386] 192.168.100.112 (192.168.100.1[192.168.100.1]): dispatching LOG_CMD command 'USERAUTH_REQUEST' to mod_log
2020-09-21 15:09:51,955 hostname proftpd[5386] 192.168.100.112 (192.168.100.1[192.168.100.1]): dispatching PRE_CMD command 'USER bob' to mod_rewrite
2020-09-21 15:09:51,955 hostname proftpd[5386] 192.168.100.112 (192.168.100.1[192.168.100.1]): dispatching PRE_CMD command 'USER bob' to mod_tls
2020-09-21 15:09:51,955 hostname proftpd[5386] 192.168.100.112 (192.168.100.1[192.168.100.1]): dispatching PRE_CMD command 'USER bob' to mod_core
2020-09-21 15:09:51,955 hostname proftpd[5386] 192.168.100.112 (192.168.100.1[192.168.100.1]): dispatching PRE_CMD command 'USER bob' to mod_core
2020-09-21 15:09:51,955 hostname proftpd[5386] 192.168.100.112 (192.168.100.1[192.168.100.1]): dispatching PRE_CMD command 'USER bob' to mod_delay
2020-09-21 15:09:51,955 hostname proftpd[5386] 192.168.100.112 (192.168.100.1[192.168.100.1]): dispatching PRE_CMD command 'USER bob' to mod_auth
2020-09-21 15:09:51,955 hostname proftpd[5386] 192.168.100.112 (192.168.100.1[192.168.100.1]): dispatching POST_CMD command 'USER bob' to mod_sql
2020-09-21 15:09:51,955 hostname proftpd[5386] 192.168.100.112 (192.168.100.1[192.168.100.1]): dispatching POST_CMD command 'USER bob' to mod_delay
2020-09-21 15:09:51,955 hostname proftpd[5386] 192.168.100.112 (192.168.100.1[192.168.100.1]): dispatching LOG_CMD command 'USER bob' to mod_sql
2020-09-21 15:09:51,956 hostname proftpd[5386] 192.168.100.112 (192.168.100.1[192.168.100.1]): dispatching LOG_CMD command 'USER bob' to mod_log
2020-09-21 15:09:51,956 hostname proftpd[5386] 192.168.100.112 (192.168.100.1[192.168.100.1]): dispatching PRE_CMD command 'PASS (hidden)' to mod_rewrite
2020-09-21 15:09:51,956 hostname proftpd[5386] 192.168.100.112 (192.168.100.1[192.168.100.1]): dispatching PRE_CMD command 'PASS (hidden)' to mod_tls
2020-09-21 15:09:51,956 hostname proftpd[5386] 192.168.100.112 (192.168.100.1[192.168.100.1]): dispatching PRE_CMD command 'PASS (hidden)' to mod_core
2020-09-21 15:09:51,956 hostname proftpd[5386] 192.168.100.112 (192.168.100.1[192.168.100.1]): dispatching PRE_CMD command 'PASS (hidden)' to mod_core
2020-09-21 15:09:51,956 hostname proftpd[5386] 192.168.100.112 (192.168.100.1[192.168.100.1]): dispatching PRE_CMD command 'PASS (hidden)' to mod_sql
2020-09-21 15:09:51,956 hostname proftpd[5386] 192.168.100.112 (192.168.100.1[192.168.100.1]): dispatching PRE_CMD command 'PASS (hidden)' to mod_delay
2020-09-21 15:09:51,956 hostname proftpd[5386] 192.168.100.112 (192.168.100.1[192.168.100.1]): dispatching PRE_CMD command 'PASS (hidden)' to mod_auth
2020-09-21 15:09:51,964 hostname proftpd[5386] 192.168.100.112 (192.168.100.1[192.168.100.1]): USER bob (Login failed): No such user found
2020-09-21 15:09:51,964 hostname proftpd[5386] 192.168.100.112 (192.168.100.1[192.168.100.1]): dispatching POST_CMD_ERR command 'PASS (hidden)' to mod_sql
2020-09-21 15:09:51,964 hostname proftpd[5386] 192.168.100.112 (192.168.100.1[192.168.100.1]): dispatching POST_CMD_ERR command 'PASS (hidden)' to mod_delay
2020-09-21 15:09:51,964 hostname proftpd[5386] 192.168.100.112 (192.168.100.1[192.168.100.1]): dispatching LOG_CMD_ERR command 'PASS (hidden)' to mod_sql
2020-09-21 15:09:51,965 hostname proftpd[5386] 192.168.100.112 (192.168.100.1[192.168.100.1]): dispatching LOG_CMD_ERR command 'PASS (hidden)' to mod_log
2020-09-21 15:09:51,965 hostname proftpd[5386] 192.168.100.112 (192.168.100.1[192.168.100.1]): dispatching LOG_CMD_ERR command 'PASS (hidden)' to mod_auth
2020-09-21 15:09:51,965 hostname proftpd[5386] 192.168.100.112 (192.168.100.1[192.168.100.1]): dispatching POST_CMD command 'USERAUTH_REQUEST' to mod_sql
2020-09-21 15:09:51,965 hostname proftpd[5386] 192.168.100.112 (192.168.100.1[192.168.100.1]): dispatching LOG_CMD command 'USERAUTH_REQUEST' to mod_sql
2020-09-21 15:09:51,966 hostname proftpd[5386] 192.168.100.112 (192.168.100.1[192.168.100.1]): dispatching LOG_CMD command 'USERAUTH_REQUEST' to mod_log
2020-09-21 15:09:51,975 hostname proftpd[5386] 192.168.100.112 (192.168.100.1[192.168.100.1]): SSH2 session closed.
```

1.3.7a
```
2020-09-21 15:08:20,862 hostname proftpd[5187] 192.168.100.112 (192.168.100.1[192.168.100.1]): dispatching POST_CMD command 'USERAUTH_REQUEST bob publickey' to mod_sql
2020-09-21 15:08:20,862 hostname proftpd[5187] 192.168.100.112 (192.168.100.1[192.168.100.1]): dispatching LOG_CMD command 'USERAUTH_REQUEST bob publickey' to mod_sql
2020-09-21 15:08:20,863 hostname proftpd[5187] 192.168.100.112 (192.168.100.1[192.168.100.1]): dispatching LOG_CMD command 'USERAUTH_REQUEST bob publickey' to mod_log
2020-09-21 15:08:20,865 hostname proftpd[5187] 192.168.100.112 (192.168.100.1[192.168.100.1]): dispatching PRE_CMD command 'USER bob' to mod_rewrite
2020-09-21 15:08:20,865 hostname proftpd[5187] 192.168.100.112 (192.168.100.1[192.168.100.1]): dispatching PRE_CMD command 'USER bob' to mod_tls
2020-09-21 15:08:20,865 hostname proftpd[5187] 192.168.100.112 (192.168.100.1[192.168.100.1]): dispatching PRE_CMD command 'USER bob' to mod_core
2020-09-21 15:08:20,865 hostname proftpd[5187] 192.168.100.112 (192.168.100.1[192.168.100.1]): dispatching PRE_CMD command 'USER bob' to mod_core
2020-09-21 15:08:20,865 hostname proftpd[5187] 192.168.100.112 (192.168.100.1[192.168.100.1]): dispatching PRE_CMD command 'USER bob' to mod_delay
2020-09-21 15:08:20,865 hostname proftpd[5187] 192.168.100.112 (192.168.100.1[192.168.100.1]): dispatching PRE_CMD command 'USER bob' to mod_auth
2020-09-21 15:08:20,865 hostname proftpd[5187] 192.168.100.112 (192.168.100.1[192.168.100.1]): dispatching POST_CMD command 'USER bob' to mod_sql
2020-09-21 15:08:20,865 hostname proftpd[5187] 192.168.100.112 (192.168.100.1[192.168.100.1]): dispatching POST_CMD command 'USER bob' to mod_delay
2020-09-21 15:08:20,865 hostname proftpd[5187] 192.168.100.112 (192.168.100.1[192.168.100.1]): dispatching LOG_CMD command 'USER bob' to mod_sql
2020-09-21 15:08:20,866 hostname proftpd[5187] 192.168.100.112 (192.168.100.1[192.168.100.1]): dispatching LOG_CMD command 'USER bob' to mod_log
2020-09-21 15:08:20,866 hostname proftpd[5187] 192.168.100.112 (192.168.100.1[192.168.100.1]): dispatching LOG_CMD command 'USER bob' to mod_delay
2020-09-21 15:08:20,866 hostname proftpd[5187] 192.168.100.112 (192.168.100.1[192.168.100.1]): dispatching PRE_CMD command 'PASS (hidden)' to mod_rewrite
2020-09-21 15:08:20,866 hostname proftpd[5187] 192.168.100.112 (192.168.100.1[192.168.100.1]): dispatching PRE_CMD command 'PASS (hidden)' to mod_tls
2020-09-21 15:08:20,866 hostname proftpd[5187] 192.168.100.112 (192.168.100.1[192.168.100.1]): dispatching PRE_CMD command 'PASS (hidden)' to mod_core
2020-09-21 15:08:20,866 hostname proftpd[5187] 192.168.100.112 (192.168.100.1[192.168.100.1]): dispatching PRE_CMD command 'PASS (hidden)' to mod_core
2020-09-21 15:08:20,866 hostname proftpd[5187] 192.168.100.112 (192.168.100.1[192.168.100.1]): dispatching PRE_CMD command 'PASS (hidden)' to mod_sql
2020-09-21 15:08:20,867 hostname proftpd[5187] 192.168.100.112 (192.168.100.1[192.168.100.1]): dispatching PRE_CMD command 'PASS (hidden)' to mod_delay
2020-09-21 15:08:20,867 hostname proftpd[5187] 192.168.100.112 (192.168.100.1[192.168.100.1]): dispatching PRE_CMD command 'PASS (hidden)' to mod_auth
2020-09-21 15:08:20,877 hostname proftpd[5187] 192.168.100.112 (192.168.100.1[192.168.100.1]): USER bob (Login failed): No such user found 
2020-09-21 15:08:20,877 hostname proftpd[5187] 192.168.100.112 (192.168.100.1[192.168.100.1]): dispatching POST_CMD command 'USERAUTH_REQUEST bob password' to mod_sql
2020-09-21 15:08:20,877 hostname proftpd[5187] 192.168.100.112 (192.168.100.1[192.168.100.1]): dispatching LOG_CMD command 'USERAUTH_REQUEST bob password' to mod_sql
2020-09-21 15:08:20,880 hostname proftpd[5187] 192.168.100.112 (192.168.100.1[192.168.100.1]): dispatching LOG_CMD command 'USERAUTH_REQUEST bob password' to mod_log
2020-09-21 15:08:20,893 hostname proftpd[5187] 192.168.100.112 (192.168.100.1[192.168.100.1]): SSH2 session closed
```

**Comment by Castaglia** [comment]
Thanks for the bug tickets (this is the same as [Bug #4411](http://bugs.proftpd.org/show_bug.cgi?id=4411).

This is a design change -- but that doesn't necessarily mean that it'll stay this way.  The behavior change occurred in [this PR](https://github.com/proftpd/proftpd/pull/976/files) trying to address this AIX-specific issue (Issue #693).

**Comment by Castaglia** [comment]
I'm trying to figure out how best to restore the previous behavior -- without regressing Issue #693.  And since this affects a lot of sites/functionality, I want to address this before the next release.

**Comment by Castaglia** [comment]
@clmerriman care to see if https://github.com/proftpd/proftpd/pull/1199 restores the expected behavior for you?

**Comment by clmerriman** [comment]
It does resolve the issue. Thanks.

**Comment by Castaglia** [comment]
I'll keep this ticket open, until that PR is merged (and probably backported).

**Comment by Castaglia** [comment]
These fixes have been merged to master, and backported to the 1.3.7 branch.  Thanks!
